client/src/test/suite/index.ts

```diff
@@ -3,42 +3,41 @@ import * as Mocha from 'mocha';
 import * as path from 'path';
 
 export function run(): Promise<void> {
-  // Create the mocha test
-  const mocha = new Mocha({
-    ui: 'tdd', // the TDD UI is being used in extension.test.ts (suite, test, etc.)
-    useColors: true, // colored output from test results
-    fullStackTrace: true,
-    reporter: 'xunit',
-    reporterOptions: {
-        // Path from the place the tests are run. In our case: /target/test-reports/client-tests.xml
-        output: '../target/test-reports/client-tests.xml',
-    },
-  });
-  mocha.useColors(true);
+    // Create the mocha test
+    const mocha = new Mocha({
+        ui: 'tdd', // the TDD UI is being used in extension.test.ts (suite, test, etc.)
+        fullStackTrace: true,
+        reporter: 'xunit',
+        reporterOptions: {
+            // Path from the place the tests are run. In our case: /target/test-reports/client-tests.xml
+            output: '../target/test-reports/client-tests.xml',
+        },
+    });
+    mocha.useColors(true);
```

Comment:
So you removed the `useColors` that was in the constructor call rather than the one that stands on its own after it? Didn’t the opposite work? Doesn’t it make more sense to keep the whole config in one place, namely the constructor call? :thinking:

Reply:
> So you removed the useColors that was in the constructor call rather than the one that stands on its own after it?

Yes, but no specific reason to do that.

> Didn’t the opposite work?

It did.

> Doesn’t it make more sense to keep the whole config in one place, namely the constructor call?

I didn't think about it at all. To me, because the constructor call and this method call are in the same place, and since both solutions have the exact same effect, I just choose randomly.

Reply:
I think it’s very confusing and gives the impression that you didn’t include it in the initial configuration object thingy for a particular reason.

Reply:
OK, done.